src/upgrader.js

```diff
@@ -205,10 +213,34 @@ class Upgrader {
     cryptoProtocol,
     direction,
     maConn,
-    muxedConnection,
+    upgradedConn,
     Muxer,
     remotePeer
   }) {
+    if (!Muxer) {
+      // Create the connection
+      maConn.timeline.upgraded = Date.now()
```

Comment:
This will miss the timeline proxy below. That should probably get moved up to the top of the function